gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerClientImpl.java

```diff
@@ -167,6 +167,7 @@ public void close() {
       try {
         actorScheduler.stop().get(15, TimeUnit.SECONDS);
       } catch (final InterruptedException | ExecutionException | TimeoutException e) {
+        Thread.currentThread().interrupt();
```

Comment:
This will mark the thread has interrupted on `TimeoutException`, or?

Reply:
yes sorry will change this